pkg/build/strategies/onbuild/entrypoint.go

```diff
@@ -24,7 +24,7 @@ func GuessEntrypoint(fs util.FileSystem, sourceDir string) (string, error) {
 		return "", err
 	}
 	for _, f := range files {
-		if f.IsDir() || !f.Mode().IsRegular() {
+		if !f.Mode().IsRegular() {
```

Comment:
Because I decided that directory is not a regular file and single should cover a both cases. Do you agree?